.github/workflows/update-sdk-snap.yml

```diff
@@ -1,4 +1,4 @@
-name: Update
+name: Update SDK Snap
 
 on:
   # Runs at 10:00 UTC every day
```

Comment:
Somewhere in this workflow you need to call the other update script.

This one runs on a cron job, if changes are made it should call the workflow to also update the content snap.

Reply:
Not sure how to do that. In the update-content-snap part, I have made it to run only when specific workflow runs

https://github.com/snapcrafters/ffmpeg-sdk/pull/9/files#diff-2e98a5de67485977aef361f5f547ec6b1ba8be7393b696ed2d8d1daf97fe499aR3-R9

and makes changes depending on the workflow. I am not sure. How to run the update workflow from within this snap. And also, if it's necessary, I think that should be ran after the sdk is published in the candidate channel.